core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -936,11 +936,19 @@ public String toString() {
   public static final PropertyKey MASTER_HEARTBEAT_INTERVAL_MS =
```

Comment:
should we use different properties for Master <-> Master heartbeat vs Worker <-> Master heartbeat?